### DISCUSSION TITLE
我真的很怀疑推荐页有没有带上用户ID

### DISCUSSION THREAD

**Discussion by Dyko-Avta** [cotaku/Bibi-Release#122] [Q&A]
上次说了这个事之后，拉黑的、反复点不感兴趣此up主的视频还是能刷出来。最离谱的是有一次把一个视频点了不感兴趣此up主，它变模糊了，点刷新，这个视频以被模糊的样子再次出现了，再刷新，它又出现了，三次的位置是不一样的，有在第一行有在第二行。
大受震撼.JPG

**Comment by cotaku** [comment]
fixed in v3.3.2.0

**Reply by 2016012036liuyun** [reply]
好像还是有这个问题，点了不想看的内容以后还是会继续刷出来，有时候还是同一条视频

**Reply by cotaku** [reply]
> 好像还是有这个问题，点了不想看的内容以后还是会继续刷出来，有时候还是同一条视频

但可以肯定的是肯定带上用户ID了，比如我的推荐视频中会有我关注的UP主：
![image](https://github.com/cotaku/Bibi-Release/assets/2822574/6394fe27-50e1-4f81-8275-66c67e4332c4)
你说的问题有可能是服务器缓存的问题？或者就是现在用的不感兴趣的API已经过时了……
